### PR TITLE
fix: use default export instead of named export

### PR DESCRIPTION
**Pull request by olafvanv**
With this you can use Webpack 5 without warnings (or even errors in case of angular 12).
All credits to @destus90, who fixed the same thing in the esri-leaflet package (https://github.com/Esri/esri-leaflet/pull/1273)

https://webpack.js.org/migrate/5/#cleanup-the-code 

> Using named exports from JSON modules: this is not supported by the new specification and you will get a warning. Instead of import { version } from './package.json' use import package from './package.json'; const { version } = package;

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "John Gravois <jgravois@esri.com> (http://johngravois.com)"
   ],
   "dependencies": {
-    "esri-leaflet": "^2.0.0",
+    "esri-leaflet": "^3.0.2",
```
Comment (gavinr) [inline package.json:11]: Is this change necessary? It does not seem related to the main change in this PR.

Comment (olafvanv) [reply package.json:11]: esri-leaflet had the same issue. I suppose you have to use an updated version, otherwise you run into the same problem. See https://github.com/Esri/esri-leaflet/pull/1273

Comment (gavinr) [reply package.json:11]: I do not see the dependency change in the PR, unless I'm missing something obvious?
image

Comment (jgravois) [reply package.json:11]: another option: https://github.com/jgravois/esri-leaflet-gp/pull/46

Comment (GVissers27) [reply package.json:11]: Any update about this? Currently running into the same issue
@gavinr you mentioned that this was fixed in esri-leaflet v3.0.2 (https://github.com/Esri/esri-leaflet/pull/1273#issuecomment-847204773) I'm guessing that's why the dependency to esri-leaflet is updated to v3.0.2

Comment (main-kun) [reply package.json:11]: Ran into the same issue, had to fork this library and force npm to resolve the package to my fork. Without fixing the version import and updating esri-leaflet to 3.0 the library cannot be used in projects built with webpack 5.

**Comment by gavinr** [comment]
Thanks for the feedback everyone. It seems like there are two things that this PR is doing:

1. [Fixing the version import](https://github.com/Esri/esri-leaflet-cluster/pull/42/files#diff-c242400d4f4ade66e9c6bd5b5177d45f0738f2b7194afe7441ed10c56bbd8efaR3-R5) - this change is valid and ok with me, as we did a very similar change in `esri-leaflet`.
2. [Changing the `esri-leaflet` dependency version in `package.json`](https://github.com/Esri/esri-leaflet-cluster/pull/42/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11) - I'm a bit hesitant to change this in the same PR as part 1, as I'm not sure it's related, and also there is an [alternative method ](https://github.com/Esri/esri-leaflet-cluster/pull/42#discussion_r751394977)proposed by @jgravois.

I'd prefer that we remove the `package.json` changes from this PR then merge this PR. Then follow-up with a separate PR/discussion around that `package.json` change (preferably to change it to `"peerDependencies": { "leaflet": "*",    "esri-leaflet": "*" ` per @jgravois). If @olafvanv wants to make this change to this PR (to remove the `package.json` change from this PR) that would be appreciated, otherwise I'll try to tackle making these changes and the additional PR shortly.

**Comment by GVissers27** [comment]
> Thanks for the feedback everyone. It seems like there are two things that this PR is doing:
> 
> 1. [Fixing the version import](https://github.com/Esri/esri-leaflet-cluster/pull/42/files#diff-c242400d4f4ade66e9c6bd5b5177d45f0738f2b7194afe7441ed10c56bbd8efaR3-R5) - this change is valid and ok with me, as we did a very similar change in `esri-leaflet`.
> 2. [Changing the `esri-leaflet` dependency version in `package.json`](https://github.com/Esri/esri-leaflet-cluster/pull/42/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11) - I'm a bit hesitant to change this in the same PR as part 1, as I'm not sure it's related, and also there is an [alternative method ](https://github.com/Esri/esri-leaflet-cluster/pull/42#discussion_r751394977)proposed by @jgravois.
> 
> I'd prefer that we remove the `package.json` changes from this PR then merge this PR. Then follow-up with a separate PR/discussion around that `package.json` change (preferably to change it to `"peerDependencies": { "leaflet": "*", "esri-leaflet": "*" ` per @jgravois). If @olafvanv wants to make this change to this PR (to remove the `package.json` change from this PR) that would be appreciated, otherwise I'll try to tackle making these changes and the additional PR shortly.

Any idea when this PR will be merged. Or are you waiting for a response?

**Comment by main-kun** [comment]
@gavinr I'm using the library right now so if it helps i can open two separate pull requests so that we don't have to wait for the author of this PR to make changes

**Comment by gavinr** [comment]
The changes in this PR have been separated out into  https://github.com/Esri/esri-leaflet-cluster/pull/49  and https://github.com/Esri/esri-leaflet-cluster/pull/50. Thanks everyone for the discussion and thanks @olafvanv  for the original PR.

**Comment by gavinr** [comment]
This has been resolved in v3.0.0  https://github.com/Esri/esri-leaflet-cluster/releases/tag/v3.0.0
